### ISSUE TITLE
You clearly don't know anything about coding.

### ISSUE THREAD

**Issue by lumonexu** [TeamRainbowDevs/Suggestion-Bot#2]
If you actually wrote the bot, then you would know that TypeScript is a programming language that compiles to JavaScript. No; It isn't a "JavaScript version of the same bot". And you even got exposed for copying code. **You** are an awful person.

You know what, I'll even self-host the bot without joining your support server. You can't do anything (or *sue* me), because you are just a random kid that doesn't know anything about coding.

This GitHub repository has no commits, and by reading the commit history, we can see that you clearly copied this code from somebody else. And the only code-related commit you've made is "credits to rainbow studios. do not remove this credits", but this clearly isn't true.

In conclusion: you are a horrible person, and you look like one of those annoying kids in my school.

By the way, you got exposed here: https://gist.github.com/Assassin-1234/8172a4e5f523f71b7f21ca204ef57b5a

**Comment by Dragonizedpizza** [comment]
you can do it without joining their support server anyway :rofl:

**Comment by Dragonizedpizza** [comment]
https://github.com/TeamRainbowDevs/Suggestion-Bot/blob/3a9a55a1a19f84c2fb0dd9eda87da33879831ae9/database/data.json#L2

He's put comments in a json file
